docs/decisions/0004-release-captain.md

```diff
@@ -1,33 +1,33 @@
 # Context and Problem Statement
 
 We want to share the responsibility of release management. Each week of the
-release, one team member will be responsible for all release-related tasks.
-This team member will be the release captain.
+release, one team member will be responsible for all release-related tasks. This
+team member will be the release captain.
 
-This document defines and agrees on the responsibilities of said release captain.
+This document defines and agrees on the responsibilities of said release
+captain.
 
 ## Considered Options
 
 ### Responsibilities of a Release Captain
 
 The release captain is responsible for the content of a release. That means
 taking appropriate measures to include all necessary fixes and features as well
-as delivering sane and functional container images.
-The release captain is also responsible for describing the changes of the new
-release in the release notes.
+as delivering sane and functional container images. The release captain is also
+responsible for describing the changes of the new release in the release notes.
 
 ### Create a Release
 
 - Bump versions in Chart (in values.yaml _and_ in Chart.yaml)
```

Comment:
Updates to `values.yaml` are no longer necessary. This has been fixed by PR s3gw-charts#46

Reply:
fixed